src/NuGet.Core/NuGet.PackageManagement/Utility/NetStandardCompatibilityUtil.cs

```diff
@@ -0,0 +1,46 @@
+﻿using NuGet.Frameworks;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace NuGet.PackageManagement
+{
+    internal static class NetStandardCompatibilityUtil
+    {
+        internal static bool DoesProjectTargetsNEt461ORGreater(NuGetFramework currentProjectFramework)
+        {
+            if (currentProjectFramework == null)
+            {
+                throw new ArgumentNullException(nameof(currentProjectFramework));
+            }
+
+            if (currentProjectFramework.IsDesktop()
```

Comment:
As discussed offline, this has been the same behavior for VS 2017 as well so we just back ported it. I think we shouldn't change anything here until we decides to explicitly not supporting ASPNet or DNX. And if we ever decides it then it should be changed for VS 2017 and not for VS 2015.